deployment/windows_menu_setup.py

```diff
@@ -4,7 +4,7 @@
 
 conda_dir = os.environ["CONDA_PREFIX"]
 menu_dir = os.path.join(conda_dir, "Menu")
-os.makedirs(menu_dir, exist_ok = True)
-shutil.copy("../doc/images/micro-sam-logo.ico", menu_dir)
+os.makedirs(menu_dir, exist_ok=True)
+shutil.copy("../doc/images/logo.ico", menu_dir)

```

Comment:
Shouldn't this be `../doc/logo/logo.ico`?

Reply:
you are correct. It should be fixed by now